src/main/scala/vexiiriscv/Param.scala

```diff
@@ -987,6 +987,7 @@ class ParamSimple() {
     plugins += new CsrRamPlugin()
     if(withPerformanceCounters) plugins += new PerformanceCounterPlugin(additionalCounterCount = additionalPerformanceCounters)
     plugins += new CsrAccessPlugin(early0, writeBackKey =  if(lanes == 1) "lane0" else "lane1")
+    plugins += new IndirectCsrPlugin(privParam.withSupervisor)
```

Comment:
This should be a disabled by default extention.
The reason is to keep VexiiRiscv small on FPGA, most optional feature are disabled. (it addup quikly ^^)